src/brod_consumer.erl

```diff
@@ -197,7 +213,8 @@ init({ClientPid, Topic, Partition, Config}) ->
   MaxBytes = Cfg(max_bytes, ?DEFAULT_MAX_BYTES),
   MaxWaitTime = Cfg(max_wait_time, ?DEFAULT_MAX_WAIT_TIME),
   SleepTimeout = Cfg(sleep_timeout, ?DEFAULT_SLEEP_TIMEOUT),
-  PrefetchCount = erlang:max(Cfg(prefetch_count, ?DEFAULT_PREFETCH_COUNT), 1),
+  PrefetchCount = erlang:max(Cfg(prefetch_count, ?DEFAULT_PREFETCH_COUNT), 0),
```

Comment:
What will happen if `PrefetchCount` is `0`?

Reply:
PrefetchCount was used as minimum size stat window size, wrongly forced to be 1.
Now PrefetchCount is only used to limit fetch ahead.
usage is: `PendingAcks.count > PrefetchCount` andalso `PendingAcks.bytes > PrefetchByets` -> then stop sending more.
`PendingAcks,count` and `bytes` are both initialised with `0`,
so the `0 > 0 -> false` result is as expected.